src/Core/tests/DeviceTests/Handlers/SearchBar/SearchBarHandlerTests.cs

```diff
@@ -120,16 +120,21 @@ public async Task MaxLengthInitializesCorrectly(int maxLength)
 			Assert.Equal(expectedText, platformText);
 		}
 
-		[Fact(DisplayName = "CancelButtonColor Initialize Correctly",
-			Skip = "This test is currently invalid https://github.com/dotnet/maui/issues/11948")]
+		[Fact(DisplayName = "CancelButtonColor Initialize Correctly"
+#if WINDOWS
+			, Skip = "This test currently fails on Windows due to https://github.com/dotnet/maui/issues/13507"
+#endif
+			)]
 		public async Task CancelButtonColorInitializeCorrectly()
 		{
 			var searchBar = new SearchBarStub()
 			{
-				CancelButtonColor = Colors.MediumPurple
+				Text = "Search",
+				Width = 200,
+				CancelButtonColor = Colors.Yellow,
 			};
 
-			await ValidateHasColor(searchBar, Colors.MediumPurple, () => searchBar.CancelButtonColor = Colors.MediumPurple);
+			await ValidateHasColor(searchBar, Colors.Yellow, () => searchBar.CancelButtonColor = Colors.Yellow, nameof(searchBar.CancelButtonColor));
```

Comment:
I think for this test we just want to use

```C#
await ValidateHasColor(searchBar, Colors.Yellow);
```

This test is validating that the color is `initialized` correctly not that it's updated.

Reply:
Thanks for noticing, I fixed this and confirmed the test still passes